src/simulation.jl

```diff
@@ -39,7 +39,7 @@ file with the parameters to use during the simulation.
 - `number_bonds_to_slice::Int=2`: the number of edges to slice.
 - `output_prefix::String="simulation_input"`: the prefix to be used for the simulation files.
 - `num_amplitudes::Union{Int64, Nothing}=nothing`: the number of amplitudes to compute.
-- `seed::Union{Int64, Nothing}=nothing`: the seed to be used for randomly selecting amplitudes to compute.
+- `seed::Union{Int64, Nothing}=nothing`: the seed to be used by flow cutter to find a tree decomposition.
```

Comment:
This seed is also used for selecting samples of amplitudes (although this is likely to change)